pkg/api/types.go

```diff
@@ -1667,6 +1667,9 @@ type NodeSpec struct {
 
 	// Unschedulable controls node schedulability of new pods. By default node is schedulable.
 	Unschedulable bool `json:"unschedulable,omitempty"`
+
+	// Untrusted indicates that the node is not trusted. By default node is trusted.
+	Untrusted bool `json:"untrusted,omitempty"`
```

Comment:
I've deliberately left out the corresponding autogenerated code updates for ease of review - I'll obviously add them if we want to merge this at some point.